google/cloud/spanner/sql_statement.h

```diff
@@ -31,6 +31,7 @@ inline namespace SPANNER_CLIENT_NS {
  * found here: <a
  * href="https://cloud.google.com/spanner/docs/lexical#case-sensitivity">Case
  * Sensitivity</a>.
+ * @note `SqlStatement` supports case-sensitive equality comparisons.
```

Comment:
Nit: I'd put a blank comment line before this `@note`

Reply:
Thanks, done. I'll merge once the tests pass